stack-macos.yaml

```diff
@@ -0,0 +1,30 @@
+resolver: nightly-2022-07-30 # GHC 9.2.3
+
+packages:
+- .
+
+docker:
+  enable: false
+  repo: psibi/alpine-haskell-stack:9.2.3v2
+
+nix:
+  # --nix on the command-line to enable.
+  packages:
+    - zlib
+    - unzip
+flags:
+  stack:
+    developer-mode: true
+
+ghc-options:
+   "$locals": -fhide-source-paths
+
+# Contains macos specific fix for https://github.com/haskell/process/issues/247
+extra-deps:
+- git: https://github.com/haskell/process.git
+  commit: b95e5fbdeb74e0cc36b6878b60f9807bd0001fa8
+- Cabal-3.6.3.0@sha256:ff97c442b0c679c1c9876acd15f73ac4f602b973c45bde42b43ec28265ee48f4,12459
```

Comment:
Is this extra-dep needed? Resolver nightly-2022-07-30 includes Cabal-3.6.3.0. https://www.stackage.org/nightly-2022-07-30/package/Cabal-3.6.3.0.

Reply:
I guess it's not required then.